crates/forge_provider/src/forge_provider/provider.rs

```diff
@@ -57,6 +59,15 @@ impl ForgeProvider {
                 HeaderValue::from_str(&format!("Bearer {api_key}")).unwrap(),
             );
         }
+
+        // Add GitHub Copilot specific header if using Copilot provider
+        if self.provider.is_copilot() {
+            headers.insert(
+                "Copilot-Integration-Id",
+                HeaderValue::from_static("forge-cli"),
```

Comment:
```suggestion
                HeaderValue::from_static("forge"),
```

Would it work with if we don't set it to `vscode-chat`?